### PR TITLE
DOC: update docstrings

### PR DESCRIPTION
**Pull request by jklenzing**
# Description

Updates docstrings as recommended by https://github.com/heliophysicsPy/standards/blob/master/standards.md

Also updates some whitespace.

## Type of change

- Documentation fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

via pytest

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``master``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.2.1] - 2020-04-13
```
Comment (jklenzing) [inline CHANGELOG.md:5]: Codacy Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/gh/sami2py/sami2py/pullRequest?prid=5352301)

Comment (jklenzing) [inline CHANGELOG.md:5]: Codacy Issue found: [[no-shortcut-reference-link] Use the trailing [] on reference links](https://app.codacy.com/gh/sami2py/sami2py/pullRequest?prid=5352301)

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,10 @@
...
+- Documentation Changes
```
Comment (jklenzing) [inline CHANGELOG.md:6]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/sami2py/sami2py/pullRequest?prid=5352301)

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+  - Updates to docstrings
```
Comment (jklenzing) [inline CHANGELOG.md:7]: Codacy Issue found: [[list-item-indent] Incorrect list-item indent: add 2 spaces](https://app.codacy.com/gh/sami2py/sami2py/pullRequest?prid=5352301)

**Comment by jklenzing** [comment]
Comments above from codacy.  Pulling out the automated webhook for this, will probably drop it altogether as #105 seems to be a better solution for style.

**Comment by jklenzing** [comment]
Thanks!

**Comment by JonathonMSmith** [comment]
I don't understand why codacy doesn't like "## [0.2.1] - 2020-04-13"

**Comment by jklenzing** [comment]
I'm not really sure how the markdown checks work

**Pull request merged**